assets/js/components/legacy-notifications/data-error.js

```diff
@@ -61,11 +69,28 @@ function getDataErrorComponent( moduleSlug, errorMessage, inGrid = false, fullWi
 
 	const reconnectURL = errorObj?.data?.reconnectURL;
 	const description = reconnectURL ? <ErrorText message={ message } reconnectURL={ reconnectURL } /> : message;
-	const cta = <CTA title={ title } description={ description } error />;
+
+	let cta = <CTA title={ title } description={ description } error />;
 
 	// This is to handle token expired error specifically.
 	if ( 'Invalid Credentials' === errorMessage ) {
```

Comment:
Looks like this may be able to be removed entirely as per @felixarntz : https://github.com/google/site-kit-wp/issues/3406#issuecomment-857116728

Reply:
@aaemnnosttv @eugene-manuilov Indeed, this can be removed.

Reply:
Ok, deleted.